src/components/Progress/README.md

```diff
@@ -1,60 +1,203 @@
 ```
 
-### Examples
+<!--/GITHUB_BLOCK-->
+
+## Stack
+
+<!--LANDING_BLOCK
 
-```js
-const progress1 = <Progress value={40} text="Completed 80 of 200" theme="success" />;
-const progress2 = <Progress value={20} theme="info" size="s" />;
-const progress3 = (
-  <Progress
+<ExampleBlock
+  code={`
+<Progress
+  stack={[
+    {theme: 'default', content: 'First', value: 25},
+    {theme: 'success', content: 'Second', value: 25},
+    {theme: 'warning', content: 'Third', value: 25},
+    {theme: 'danger', content: 'Fourth', value: 25},
+  ]}
+/>
+<Progress text="Progress with custom colors"
+  stack={[
+    {color: '#6e5d8c', value: 33.333333333333336},
+    {color: '#5b785b', value: 33.333333333333336},
+    {color: '#956069', value: 33.333333333333336},
+  ]}
+/>
+`}
+>
+<div style={{width: '30%'}}>
+  <UIKit.Progress
     stack={[
-      {value: 40, color: 'red'},
-      {value: 10, color: 'blue'},
+      {theme: 'default', content: 'First', value: 25},
+      {theme: 'success', content: 'Second', value: 25},
+      {theme: 'warning', content: 'Third', value: 25},
+      {theme: 'danger', content: 'Fourth', value: 25},
     ]}
   />
-);
+</div>
+<div style={{width: '30%'}}>
+  <UIKit.Progress text="Progress with custom colors"
+    stack={[
+      {color: '#6e5d8c', value: 33.333333333333336},
+      {color: '#5b785b', value: 33.333333333333336},
+      {color: '#956069', value: 33.333333333333336},
+    ]}
+  />
+</div>
+</ExampleBlock>
+
+LANDING_BLOCK-->
+
+<!--GITHUB_BLOCK-->
+
+```tsx
+<Progress stack={[
+  {theme: 'default', content: 'First', value: 25},
+  {theme: 'success', content: 'Second', value: 25},
+  {theme: 'warning', content: 'Third', value: 25},
+  {theme: 'danger', content: 'Fourth', value: 25},
+]} />
+<Progress text="Progress with custom colors" stack={[
+  {color: '#6e5d8c', value: 33.333333333333336},
+  {color: '#5b785b', value: 33.333333333333336},
+  {color: '#956069', value: 33.333333333333336},
+]} />
 ```
+
+<!--/GITHUB_BLOCK-->
+
+## Properties
+
+| Name            | Description                                                                                                       |                  Type                   | Default     |
+| :-------------- | :---------------------------------------------------------------------------------------------------------------- | :-------------------------------------: | :---------- |
+| className       | HTML `class` attribute                                                                                            |                `string`                 |             |
+| colorStops      | Array of sections themes and values                                                                               | `Array<{theme: string; stop: number;}>` |             |
+| colorStopsValue | Alternative value of `colorStops`. Available range: 0 to 100.                                                     |                `number`                 |             |
```

Comment:
We need section describing this behaviour

Reply:
<img width="400" alt="Screenshot 2023-08-25 at 19 56 31" src="https://github.com/gravity-ui/uikit/assets/22904809/2992427f-618a-4a18-9ec5-d46124f1972e">
<img width="400" alt="Screenshot 2023-08-25 at 19 56 43" src="https://github.com/gravity-ui/uikit/assets/22904809/4c3e038e-7d8b-4b83-b580-b02bb3b21666">